src/selectivity-base.js

```diff
@@ -207,6 +211,12 @@ $.extend(Selectivity.prototype, EventDelegator.prototype, {
         return this.$el.find(selector);
     },
 
+    _blurClose: function() {
```

Comment:
Why not simply call this method `_blur`, to be more consistent with `_mouseover`, etc.. Also, please move the method down so it will be with the other private methods (I know I'm a bit pedantic about it, I even sort them alphabetically :p).